build/dependencies.props

```diff
@@ -4,9 +4,13 @@
     <AspNetIntegrationTestingVersion>0.4.0-*</AspNetIntegrationTestingVersion>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
     <AutoMapperVersion>5.2.0</AutoMapperVersion>
+    <ImmutableCollectionsVersion>1.3.1</ImmutableCollectionsVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
```

Comment:
For our src/ projects, it's still needed if you have netstandard1.x project. In this repo, it doesn't matter either way as we only have samples.